package.json

```diff
@@ -91,6 +91,7 @@
     "sinon-chai": "^2.7.0",
     "spec-xunit-file": "0.0.1-3",
     "supertest": "^1.1.0",
+    "supertest-as-promised": "^4.0.0",
```

Comment:
You don't need this. Just use `this.request()`...`.end()`  - That will return a promise.